amqp_influxdb/__main__.py

```diff
@@ -20,13 +20,21 @@
     InfluxDBPublisher,
     AMQPTopicConsumer,
     BATCH_SIZE,
-    MAX_BATCH_DELAY)
+    MAX_BATCH_DELAY,
+    BROKER_PORT_SSL,
+    BROKER_PORT_NO_SSL,
+)
 
 
 def parse_args():
     parser = argparse.ArgumentParser()
     parser.add_argument('--amqp-hostname', required=False,
                         default='localhost')
+    parser.add_argument('--amqp-username', required=True)
```

Comment:
it can still be required with validation only taken place at the manager blueprint level. all the other places should support guest:guest by default as this is what developers will most likely already have. we should support easy development process as much as possible